src/NHSD.BuyingCatalogue.Ordering.Domain/Orders/OrderErrors.cs

```diff
@@ -16,5 +16,46 @@ public static ErrorDetails OrderOrganisationIdRequired()
         {
             return new ErrorDetails("OrganisationIdRequired", nameof(Order.OrganisationId));
         }
+
+        public static ErrorDetails ContactNameRequired(string fieldName)
+        {
+            return new ErrorDetails(fieldName+"Required", fieldName);

```

Comment:
Again, we said we werent adding validation to this checkin, this shouldnt be here